docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-helm-addon.md

```diff
@@ -49,6 +49,7 @@ You can extend the list of integrations by adding Helm charts from a public or p
   pack:
    namespace: your_namespace_here
   ``` 
+  Optionally, you can add configurable layer values and specify the layer [install order](./create-pack-addon.md#install-order).
```

Comment:
@stenwt what is a configurable layer? Maybe we can expand a bit more on what that means.